client/reader/components/favicon/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+function Favicon( props ) {
+	const addFallbackImage = ( e ) => {
+		e.target.src = 'https://wordpress.com/favicon.ico';
```

Comment:
Great suggestions.
> Would it make sense to use an image included in Calypso (https://github.com/Automattic/wp-calypso/tree/master/static/images) rather than relying on an asset deployed/maintained elsewhere? Would also allow us to avoid the hard-coded WordPress.com reference and instead use a relative path.

> At least in the case of the placeholder, can we avoid the low-res favicon in favor of an SVG or similarly eyeball-friendly format?

I updated the load fallback to use `Gridicon`

> Along the same lines, is there a better asset we can retrieve for site icons instead of the .ico?

I updated the main image to be the site's icon (blavatar) and only fallback to favicon.